### DISCUSSION TITLE
TeamTalk Admin authority

### DISCUSSION THREAD

**Discussion by turalmemmedov123** [BearWare/TeamTalk5#2413] [General]
HELLO. I have a request from you. For some reason, and for people's safety, I want admin accounts on my Teamtalk server to not be able to monitor people's voices, messages, etc. I want them to remain administrators, but they should never be able to follow People's voices and messages. I can't even do this myself. Can you help me with this?

**Comment by bear101** [comment]
This is not a feature we plan to implement. An administrator is like "root" user and they can do _everything_

**Comment by amirmahdifard** [comment]
you can not make users admins and just give them user rights instead. actually intercept is something useful for admins
